backend/backup_media.js

```diff
@@ -0,0 +1,151 @@
+require('dotenv-flow').config();
+const mysql = require('mysql2');
+const fs = require('fs');
+const crypto = require('crypto');
+const http = require('http');
+const https = require('https');
+const sizeOf = require('image-size');
+
+const MEDIA_TYPE_IMG = 'img';
+const MEDIA_TYPE_VIDEO = 'video';
+
+const config = {
+    host: process.env.MYSQL_HOST || "localhost",
+    port: parseInt(process.env.MYSQL_PORT) || 3306,
+    user: "root",
+    password: process.env.MYSQL_ROOT_PASSWORD || "orbitar",
+    database: process.env.MYSQL_DATABASE || "orbitar_db",
+    dir: './media-backup',
+    dbFile: './media-backup/index.csv'
+};
+
+const sleep = function(ms) {
+  return new Promise(function(resolve) {
+    setTimeout(resolve, ms);
+  });
+}
+
+const download = function(url, dest, cb) {
+  if (url.match(/^\/\//)) {
+    url = 'https:' + url;
+  }
+  try {
+    const file = fs.createWriteStream(dest);
+    let loader = http;
+    if (url.includes('https://')) {
+      loader = https;
+    }
+    loader.get(url, function(response) {
+      response.pipe(file);
+      file.on('finish', function() {
+        file.close(cb);
+      });
+    }).on('error', function(err) {
+      console.trace(err);
+      fs.unlinkSync(dest);
+      if (cb) cb(err.message);
+    });
```

Comment:
I think you need to call `.end()` to send the request (not sure if it's absence cause any actual issues).

Reply:
> I think you need to call .end() to send the request (not sure if it's absence cause any actual issues).

Not sure I understand. Do you mean `http.get().send()`? If yes, then no, it's not needed